static/js/commentIcons.js

```diff
@@ -1,222 +1,216 @@
-var $ = require('ep_etherpad-lite/static/js/rjquery').$;
-var _ = require('ep_etherpad-lite/static/js/underscore');
-var commentBoxes = require('ep_comments_page/static/js/commentBoxes');
+'use strict';
+
+const commentBoxes = require('ep_comments_page/static/js/commentBoxes');
 
 // Indicates if Etherpad is configured to display icons
-var displayIcons = function() {
-  return clientVars.displayCommentAsIcon
-}
+const displayIcons = () => clientVars.displayCommentAsIcon;
 
 // Easier access to outer pad
-var padOuter;
-var getPadOuter = function() {
+let padOuter;
+const getPadOuter = () => {
```

Comment:
In general I quite like ``function(){`` so can we not be overly strict about this?  I prefer the syntax and I think it's more descriptive, just my personal opinion.  Can merge w/ your changes in, I don't hate them, I just don't mind either approach 👍

Reply:
`function` and arrow functions [do not behave the same](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions), so it's not just a matter of visual preference.

Good style is to only use `function` for class methods (though see [ES6 method shorthand](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions)) and arrow functions for everything else. There are exceptions, such as callbacks passed to [jQuery's `.each()` method](https://api.jquery.com/each/) and certain event handlers.